src/test/ui/consts/projection_qualif.stderr

```diff
@@ -1,8 +1,11 @@
-error[E0017]: references in constants may only refer to immutable values
+error[E0658]: references in constants may only refer to immutable values
```

Comment:
Make the test corresponding to this stderr file a test that checks both with the feature flag and without by employing https://github.com/rust-lang/rust/pull/66524